code/modules/mob/living/carbon/human/species/station/station.dm

```diff
@@ -525,7 +525,10 @@
 	var/sprint_charge_factor = 0.65
 
 datum/species/machine/handle_post_spawn(var/mob/living/carbon/human/H)
-	H.gender = NEUTER
+	if(!H.species == "Shell Frame")
```

Comment:
`mob.species` is a datum, it cannot be compared to a string. Further, you're comparing the inversion of it, which is probably going to make it return false regardless. It's _also_ bad OOP to do any of this.

Probably the best way to do this is to use a variable:
```dm
/datum/species/machine
	// .. Other variables
	var/neuter_only = TRUE

/datum/species/machine/handle_post_spawn(var/mob/living/carbon/human/H)
	if (neuter_only)
		H.gender = NEUTER

	. = ..()

	check_tag(H, H.client)

[in machine_subspecies.dm]
/datum/species/machine/shell
	// Other variables
	neuter_only = FALSE
```